incubator/distribution/README.md

```diff
@@ -97,6 +101,48 @@ The following table lists the configurable parameters of the distribution chart
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
 
+### Ingress and TLS
+To get Helm to create an ingress object with a hostname, add these two lines to your Helm command:
+```
+helm install --name distribution \
+  --set ingress.enabled=true \
+  --set ingress.hosts[0]="distribution.company.com" \
+  --set distribution.service.type=NodePort \
+  incubator/distribution
+```
+
+If your cluster allows automatic creation/retrieval of TLS certificates (e.g. [kube-lego](https://github.com/jetstack/kube-lego)), please refer to the documentation for that mechanism.
```

Comment:
Replace `kube-lego` which is depreciated with the `[cert-manager](https://github.com/jetstack/cert-manager)`

Reply:
@rimusz done